src/components/availability-panel/drag-drop-context/DroppableComponent.tsx

```diff
@@ -28,9 +97,62 @@ const DroppableComponent: FC<droppableComponent> = ({
       <Droppable droppableId="members" isCombineEnabled={isTaskOnDrag}>
         {(provided) => (
           <div ref={provided.innerRef}>
-            {idleMembers.map((member, index) => (
-              <DraggableComponent draggableId={member} index={index} key={member} />
-            ))}
+            {idleMembers.filter((memberItem) => {
+              if (searchTermMember === "") {
+                return true;
+              }
+              if (
+                memberItem
+                  .toLowerCase()
+                  .includes(searchTermMember.trim().toLowerCase())
+              ) {
+                return true;
+              }
+              return false;
+            }).length === 0 ? (
+              <NotFoundError message="No idle members found" />
+            ) : (
+              idleMembers
+                .filter((memberItem) => {
+                  if (searchTermMember === "") {
+                    return true;
+                  }
+                  if (
+                    memberItem
+                      .toLowerCase()
+                      .includes(searchTermMember.trim().toLowerCase())
+                  ) {
+                    return true;
+                  }
+                  return false;
```

Comment:
I am observing a repeated pattern here, could we create a function to address this please